barretenberg/sol/src/honk/keys/EcdsaHonkVerificationKey.sol

```diff
@@ -15,72 +15,72 @@ library EcdsaHonkVerificationKey {
             logCircuitSize: uint256(16),
             publicInputsSize: uint256(6),
             ql: Honk.G1Point({
-                x: uint256(0x0b1acdcf739e1e6c27df046577122a292a77f4fcdf8056d8b8ae12f105d3a888),
-                y: uint256(0x145dad3bdd9a262411aaa657129df49dbf44a63f510e9ab8191622c643ebd9bd)
+                x: uint256(0x051ccdb8069f35f4ef85ad098e95681736a7bed10a7bee1b76a506235dc0b579),
```

Comment:
I've noticed that these vks change magically (maybe via the build process?) but they should based on the changes you made